platforms/common/glfwApp.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "platform.h"
+#include "tangram.h"
+#include <GLFW/glfw3.h>
+#include <memory>
+
+namespace Tangram {
+
+namespace GlfwApp {
+
+    void create(std::shared_ptr<Platform> platform, std::string sceneFile, int width, int height);
+    void destroy();
+
+    void run();
+    void stop(int);
+
+    void mouseButtonCallback(GLFWwindow* window, int button, int action, int mods);
```

Comment:
We could remove those from the header definition, so we would include `<GLFW/glfw3.h>` from implementation file only. Not that harmful though.